raspberry_pi/app/camera/camera_factory.py

```diff
@@ -42,6 +42,7 @@ def get_num_cameras(identity: Identity) -> int:
                 | Identity.SWERVE_LEFT
                 | Identity.FUNNEL
                 | Identity.DEV # has one v2 camera at the moment
+                | Identity.DEV2 # has one v2 camera at the moment
```

Comment:
when you notice comments that are wrong, you should change or remove them, definitely not just blindly copy them.